src/commands/ExecCommand.js

```diff
@@ -18,17 +18,19 @@ export const builder = {
     group: "Command Options:",
     describe: "Bail on exec execution when the command fails within a package",
     type: "boolean",
-    default: true,
+    default: undefined,
```

Comment:
I was noodling about using yargs' [`config()`](https://github.com/yargs/yargs/blob/master/docs/api.md#config) method (as described [here](https://github.com/yargs/yargs/blob/master/docs/advanced.md#rc-files)) with the parsed lerna.json stuff, and it's certainly more elegant (and in practice the same idea as you propose).

The chicken & egg problem strikes us here, because the arg parsing (as currently constructed) has already finished by the time we read lerna.json and filter the command-specific durable config, etc. We should probably extract that chunk (config resolution and filtering) anyway, but it's a bit complex to contemplate.